src/library_webgl.js

```diff
@@ -3599,6 +3601,59 @@ var LibraryGL = {
     GLctx.sampleCoverage(value, !!invert);
   },
 
+  glMultiDrawArraysWEBGL: function(mode, firsts, firstsOffset, counts, countsOffset, drawcount) {
+    var firstsList = HEAP32.subarray(firsts >> 2, (firsts >> 2) + drawcount);
+    var countsList = HEAP32.subarray(counts >> 2, (counts >> 2) + drawcount);
+    GLctx.multiDrawWebgl['multiDrawArraysWEBGL'](
+      mode,
+      firstsList,
+      firstsOffset,
+      countsList,
+      countsOffset,
+      drawcount);
```

Comment:
Looking back at the multi-draw extension:
https://www.khronos.org/registry/webgl/extensions/WEBGL_multi_draw/
and the ANGLE extension which it exposes:
https://chromium.googlesource.com/angle/angle/+/master/extensions/ANGLE_multi_draw.txt

I just realized that the C signatures in this pull request need to be updated - pairs like "firsts / firstsOffset" should be just a single pointer in the C signature. See comment below for an example. All of the entry points in this PR must be fixed.

This one should be written this way (please test this):

```
glMultiDrawArraysANGLE: function(mode, firsts, counts, drawcount) {
    GLctx.multiDrawWebgl['multiDrawArraysWEBGL'](
      mode,
      HEAP32,
      firsts >> 2,
      HEAP32,
      counts >> 2,
      drawcount);
```

("firsts" and "counts" are pointers into the WASM heap, which are just 32-bit numbers representing byte offsets. They need to be converted into offsets into an Int32Array.)